packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -595,11 +595,7 @@
     @include mobile {
         display: grid;
         grid-template-areas: 'a b';
-        grid-column-gap: 0.5rem;
-
-        &--accumulator {
-            display: block;
-        }
+        grid-column-gap: 0.8rem;
```

Comment:
@yauheni-kryzhyk-deriv this will affect other contract types, please apply the change to &--accumulator only🙂